src/index.js

```diff
@@ -1,8 +1,9 @@
 import './style.css';
+import '../node_modules/@fortawesome/fontawesome-free/js/all.js';
 
 const todoHTML = document.querySelector('.todo_list');
 
-let todoTasks = [
+const todoTasks = [
   {
     desctiption: 'string',
     completed: 'bool',
```

Comment:
Good job so far 👍 but there are still some minute issues to address before you are good to go.
- [ ] The `completed` key should be a boolean, so either it should have a value of true or have a value of false. It would be great if you could apply that.
- [ ] Also the index key should be the number of the item in the array. For example, the first item should have an index of 1, the second an index of 2 and so on. Kindly address this issue 😄

Reply:
Hi, thank you. is it necessary to use indexing fom 1 or I can use 0 as the first index?